_rules/target-size-empty-kj4tr0.md

```diff
@@ -0,0 +1,159 @@
+---
+id: kj4tr0
+name: Interactive component has no clickable area
+rule_type: atomic
+description: |
+  This rule checks that elements that can receive pointer events have an actual size reduced to 0.
+accessibility_requirements:
+  wcag21:2.5.5: # Target size (enhanced) (AAA)
+    secondary: 'This success criterion is **less strict** than this rule. This is because the rule does not consider the size of some elements. Some of the failed examples may satisfy this success criterion.'
+  wcag22:2.5.8: # Target Size (Minimum) (AA)
+    secondary: 'This success criterion is **less strict** than this rule. This is because this criterion has a lower size requirement. Some of the failed examples may satisfy this success criterion.'
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  test_assets: Map Image by <a href="https://www.freepik.com/free-vector/black-white-town-navigation-map_5663353.htm">Freepik</a>
+---
+
+## Applicability
+
+This rule applies to any [HTML element][namespaced element] which [can be targeted by a pointer event][].
+
+## Expectation
+
+Each test target has an empty [clickable area][], and its [clickable area][] cannot be made non-empty through scrolling.
+
+## Assumptions
+
+- This rule assumes that [focusable][] `widget` are effectively clickable. If a widget is [focusable][] without being clickable, it may fail this rule while [Success Criterion 2.5.5 Target Size (enhanced)][sc255] and [Success Criterion 2.5.8 Target Size (minimum)][sc258] are satisfied.
+
+## Accessibility Support
+
+Hit testing isn't properly defined, and this has been an [issue in the CSS specification](https://github.com/w3c/csswg-drafts/issues/2325) for years. Therefore, different User Agents may perform it differently, resulting in different [clickable areas][clickable area] for the same element. As of February 2024, the ACT rules Community Group is not aware of actual cases resulting in significantly different [clickable areas][clickable area].
+
+## Background
```

Comment:
I need some more background here. I'm not sure I understand what this rule is trying to do or really even how the S.C. are being applied.